tasks/managed/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -91,6 +91,7 @@ spec:
         else
           iib_service_account_secret="iib-service-account-prod"
         fi
+        publishing_credentials=$(jq -r '.fbc.publishingCredentials' "$DATA_FILE")
```

Comment:
No default here? If it's not there, you would get `null` - probably not desirable.

Reply:
I think in the past we would hardcode the default here using `//`. But another option would be to only pass the parameter if set in the data file. But probably the former is easier and also consistent with other places.

Reply:
default added.